src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -175,7 +175,7 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                             }
                             else
                             {
-                                _logger.LogWarning(0, ex, $"Unable to bind to {parsedAddress.ToString()} on the IPv6 loopback interface.");
```

Comment:
What if the error isn't `-4089 EAFNOSUPPORT`? Should we special case just that error code and log the exception otherwise? If it is something else, it would be good to know.